service/src/main/java/uk/gov/hmcts/reform/pebble/TodayFilter.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.pebble;
+
+import com.mitchellbosecke.pebble.extension.Filter;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class TodayFilter implements Filter {
+    @Override
+    public List<String> getArgumentNames() {
+        return null;
+    }
+
+    @Override
+    public Object apply(Object input, Map<String, Object> args) {
```

Comment:
why do you have parameters here? you dont use any of them?

Reply:
Param is needed

Reply:
even input? becuase you dont use that either

Reply:
Yes unfortunately it's implementation detail of Pebble Filter.